core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.component.html

```diff
@@ -64,6 +64,7 @@ <h3>
       <div class="thumbnail-editor protractor-test-subtopic-thumbnail protractor-test-photo-button create-new-subtopic-input-field">
         <strong>Thubmnail Image*</strong>
         <oppia-thumbnail-uploader (updateFilename)="updateSubtopicThumbnailFilename($event)"
+                                  [useLocalStorage]="false"
```

Comment:
Just to check, subtopic images are stored under the topic entity, right? (or else, this needs to be true).

Reply:
yeah, they are stored under entity type - topic.